src/front-door/azext_front_door/custom.py

```diff
@@ -261,24 +271,40 @@ def update_fd_frontend_endpoints(instance, host_name=None, session_affinity_enab
     return instance
 
 
-def configure_fd_frontend_endpoint_https(cmd, resource_group_name, front_door_name, frontend_endpoint_name,
-                                         disable=None, protocol=None, secret_name=None, secret_version=None,
-                                         certificate_type=None, certificate_source=None, vault=None):
-    if disable:
-        return cf_fd_frontend_endpoints(cmd.cli_ctx, None).disable_https(resource_group_name, front_door_name,
-                                                                         frontend_endpoint_name)
-    # if not being disabled, then must be enabled
+def configure_fd_frontend_endpoint_https_disable(cmd, resource_group_name, front_door_name, item_name):
+    return cf_fd_frontend_endpoints(cmd.cli_ctx, None).disable_https(resource_group_name, front_door_name,
```

Comment:
Seems this could also be done with 
`g.command('disable-https', 'disable_https')`

Reply:
Tried it. Failing with below error
The operation 'azext_front_door.vendored_sdks.operations.front_doors_operations#FrontDoorsOperations.disable_https' is invalid.
Traceback (most recent call last):
  File "E:\azure-cli-extensions\env\lib\site-packages\azure\cli\core\__init__.py", line 498, in get_op_handler
    op = getattr(op, part)
AttributeError: type object 'FrontDoorsOperations' has no attribute 'disable_https'

Reply:
Ah, you would need to add `client=cf_fd_frontend_endpoints` in the command definition.  Otherwise it defaults to the standard frontdoor one.